### ISSUE TITLE
Syntax error with imports

### ISSUE THREAD

**Issue by snailuj** [sqreen/PyMiniRacer#72]
### Steps to reproduce
I love the idea of using something like PyMiniRacer to do server-rendering of React components without needing to manage a Node server. But I'm stuck on the very first steps ... importing React. I'm not sure if this is a bug or a problem in my understanding. If so perhaps there is a missing step in the documentation?

I'm unable to do any JS imports (ES6 or ES5). Do I need to upgrade V8 or build it from source perhaps?

In virtual environment run `pip install py_mini_racer`, then:
```
(flask-mini-racer-rGW75Ozx) ➜  flask-mini-racer git:(master) ✗ python
Python 3.5.3 (default, Sep 27 2018, 17:25:39)
[GCC 6.3.0 20170516] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from py_mini_racer import py_mini_racer
>>> ctx = py_mini_racer.MiniRacer()
>>> ctx.eval("import { createElement } from 'react';")
```

Or, for ES5:

```
>>> ctx.eval("const React = require('react')");
```

### Expected behavior
There should be no syntax error

### Actual behavior
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/julian/.local/share/virtualenvs/flask-mini-racer-rGW75Ozx/lib/python3.5/site-packages/py_mini_racer/py_mini_racer.py", line 146, in eval
    python_value = res.contents.to_python()
  File "/home/julian/.local/share/virtualenvs/flask-mini-racer-rGW75Ozx/lib/python3.5/site-packages/py_mini_racer/py_mini_racer.py", line 272, in to_python
    raise JSParseException(msg)
py_mini_racer.py_mini_racer.JSParseException: b'SyntaxError: Unexpected token {'
```

Similar syntax error with ES5 'require':
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/julian/.local/share/virtualenvs/flask-mini-racer-rGW75Ozx/lib/python3.5/site-packages/py_mini_racer/py_mini_racer.py", line 146, in eval
    python_value = res.contents.to_python()
  File "/home/julian/.local/share/virtualenvs/flask-mini-racer-rGW75Ozx/lib/python3.5/site-packages/py_mini_racer/py_mini_racer.py", line 275, in to_python
    raise JSEvalException(msg.decode('utf-8', errors='replace'))
py_mini_racer.py_mini_racer.JSEvalException: ReferenceError: require is not defined
ReferenceError: require is not defined
    at <anonymous>:1:15
```

### System configuration
**PyMiniRacer version**: 0.1.17

**Python version**: 3.5.3 (with pipenv)

**Comment by aviat** [comment]
Hi @snailuj,
PyMiniRacer isn’t Node. It won’t translate a require into file system calls.
In order to make this work you’ll have to bundle React in one single file using eg webpack, and then eval this file in a PyMiniRacer context.
Happy to see any more elaborate documentation you could produce about that!

**Comment by snailuj** [comment]
Hello, thanks so much for your reply and apologies for bothering you with
that. I confess to not understanding all the moving parts - I'm a newbie
with Node and hadn't realised that it was essentially a whole runtime.

I tried the webpack approach, which makes total sense now that I know more,
but when I do that for my use case (server rendering a hello world
component with `ReactDOMServer.renderToString()`) my bundled code returns
only an empty javascript object from `ctx. eval`.

I've looked into the React source and noticed that `renderToString` is
actually part of a module `ReactServerNode`. Maybe it has some Node
dependencies as well, hence the lack of useful output.

This is all based on a cursory browse of the React source. I've done a bit
of searching for others using `renderToString` outside of the Node
environment, but haven't seen much. I hardly imagine I'm the first to try
doing it. Possibly there's some more fundamental impediment, which would be
a shame.

I'm part of a startup looking to spike out a server rendered prototype ASAP
for a large rewrite of our flagship product, but adding Node to our stack
is undesirable from a package management perspective (pip is worry enough!).

I will investigate more after the holidays and report back, but it's safe
to say any findings will be out of scope of the issue in this thread.

Happy holidays and thanks for the quick reply!
